### PR TITLE
Tablets and fixing errors

### PR DESCRIPTION
**Pull request by eGzIT**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE HTML>
+<html>
+<head>
```
Comment (d3usss) [inline index.html:3]: you missing tab here, the same with the body on 14 line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main>
+        <!--About school-->
+        <div class="container-fluid">
+                <div class="container">
+                    <div class="row about-top">
```
Comment (d3usss) [inline index.html:57]: In most cases is good :). but I believe row should be the only class better move about-top to child div.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+                        <div class="about__class">
+                            <div class="col-sm-12 col-lg-4">
+                                <img src="img/online-class.svg" class="img-fluid rounded mx-auto d-block">
+                                <h1 class="text-center">JAK WYGLĄDAJĄ ZAJĘCIA</h1>
+                                <p>Zajęcia odbywają się online, co daje mi bardzo wiele możliwości prowadzenia ich w interaktywny i ciekawy sposób. Na początku ustalimy między innymi z jakich aplikacji oraz materiałów będziemy korzystać, kto i w jaki sposób będzie prowadził notatki oraz jak będziemy sprawdzać twoje postępy w nauce. </p>
+                            </div>
+                        </div>
+
+                        <div class="about__effort">
```
Comment (d3usss) [inline index.html:81]: is this div is needed? can we combine about__efort with chiald div

File: index.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+                        </form>
+                    </div>
+
+                    <div class="col-sm-12 data">
+                        <h5>Dane kontaktowe:</h5>
+                        <p>Magdalena Przybyszewska</p>
+                        <h5>mail:</h5><p>kontakt@englishspark.pl</p>
+                        <h5>skype:</h5><p>live:.cid.ea1459dabc2ab736</p>
+                        <h5>tel:</h5><p>784 261 357</p>
+                        <img src="img/owl.png" class="img-fluid rounded mx-auto d-block">
```
Comment (d3usss) [inline index.html:187]: can we use the picture element because is better peroformnce for smaller and slower devices

File: index.html (hunk 1 above)
Comment (eGzIT) [reply index.html:3]: If I understood correctly, it's done.

File: index.html (hunk 2 above)
Comment (eGzIT) [reply index.html:57]: Done,

File: index.html (hunk 3 above)
Comment (eGzIT) [reply index.html:81]: Done.

File: index.html (hunk 4 above)
Comment (eGzIT) [reply index.html:187]: I need your help with that.

**Pull request merged**